src/protocols-table.js

```diff
@@ -36,9 +36,11 @@ Protocols.table = [
   [132, 16, 'sctp'],
   // all of the below use varint for size
   [302, 0, 'utp'],
-  [421, Protocols.lengthPrefixedVarSize, 'ipfs'],
-  // preferred name for 421 (added below ipfs, p2p takes precedence during table population)
   [421, Protocols.lengthPrefixedVarSize, 'p2p'],
+  // preferred name for 421 (added below p2p, ipfs takes precedence during table population)
```

Comment:
This comment isn't correct anymore.

Reply:
Oh wait. It is correct.

Reply:
I did an update because I saw it wasn't correct when I made the change and fixed the comment. The push may have just taken a little bit as I'm on a train 🚆.

Reply:
I find it confusing (as you can see on my comments). Why not putting a comment before the `p2p` saying:

```
// `p2p` is the preferred name for 421
```

And before `ipfs`:

```
// `ipfs` is intentionally below `p2p` so that `ipfs` takes precedence during table population
// this enables us ...
```

Perhaps in nicer English, but you get the point :)